src/PIL/ImageFont.py

```diff
@@ -927,7 +932,10 @@ def load_path(filename: str | bytes) -> ImageFont:
             return load(os.path.join(directory, filename))
         except OSError:
             pass
-    msg = "cannot find font file"
+    msg = f"cannot find font file '{filename}' in sys.path"
```

Comment:
```suggestion
    msg = f'cannot find font file "{filename}" in sys.path'
```
I've realised that both this and the second half of the message use quotes, and I think it's better if they use the same type.

You might ask why using double quotes in `ImageFont.load("...")` is good. I think you will gather from our codebase that this is the style we have chosen.